### PR TITLE
fixed conditionals for s3 upload enabling

### PR DESCRIPTION
**Pull request by kave**
@kurtw

### CODE REVIEW

**Comment by kurtw** [comment]
How is this different from the older code?

**Comment by kave** [comment]
@kurtw os.environ.get() returns a string not a boolean so the if statement was always true. needed to explicitly compare it to a String to get the correct behaviour

File: cfgov/cfgov/settings/base.py [hunk 1]
```diff
@@ -262,7 +262,7 @@
 PDFREACTOR_LIB = os.environ.get('PDFREACTOR_LIB', '/opt/PDFreactor/wrappers/python/lib')
 
 # S3 Configuration
-if os.environ.get('S3_ENABLED', False):
+if os.environ.get('S3_ENABLED', 'False') == 'True':
```
Comment (kurtw) [inline cfgov/cfgov/settings/base.py]: what about `getattr`?

Comment (kave) [inline cfgov/cfgov/settings/base.py]: @kurtw ? not sure how that helps

Comment (kurtw) [inline cfgov/cfgov/settings/base.py]: Sorry I meant `eval()` but now that I'm thinking about it that's probably a bad idea, security wise.

**Comment by kurtw** [comment]
:+1:

**Comment by kurtw** [comment]
:+1:

**Pull request merged**